app/forms/keywords_form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class KeywordsForm
+  include ActiveModel::Model
+
+  validates_with KeywordsFormValidator
+
+  attr_reader :file, :keyword_ids
+
+  def initialize(user)
+    @user = user
+  end
+
+  def save(file)
+    @file = file
+
+    return false unless valid?
+
+    begin
+      keyword_records = parse_keywords.map { |keyword| keyword_record(keyword) }
+      # rubocop:disable Rails::SkipsModelValidations
+      @keyword_ids = Keyword.insert_all(keyword_records).map { |hash| hash['id'] }
```

Comment:
I don't see we are using `keyword_ids` anywhere 🤔 , if we don't use it how about removing it?

Reply:
We will use `keyword_ids` to SearchKeywordsJob, I mistakely commented in the `KeywordsController`, will update it

Reply:
Fixed in [`f84cf8d` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/f84cf8dbc89dd5a6e5ac9bb1503f04ae0923f7c4)